common/djangoapps/microsite_configuration/backends/base.py

```diff
@@ -0,0 +1,140 @@
+"""
+Microsite configuration backend module.
+
+Contains the base class for microsite backends.
```

Comment:
It would be helpful if we described the base implementation here as well (ie, uses filesystem) in addition to the information included in the docstring for the actual class.